brion/synapse.cpp

```diff
@@ -357,20 +395,17 @@ class Synapse : public boost::noncopyable
         lunchbox::ScopedWrite mutex( detail::_hdf5Lock );
         SilenceHDF5 silence;
 
-        BOOST_FOREACH( UnmappedFiles::value_type& entry, _unmappedFiles )
+        // this trial-and-error is the 'fastest' path found
+        BOOST_FOREACH( const std::string& candidate, _fileNames )
         {
-            const std::string& candidate = entry.first;
-            H5::H5File& file = entry.second;
-            // keeping the files open 'saves' some time
```

Comment:
No, it's not. I changed the code to not keep the files open because otherwise the process will run out of file descriptors.